### ISSUE TITLE
Screenshots taken in a afterEach hook are not being displayed in allure report

### ISSUE THREAD

**Issue by rodolforodriguess** [Shelex/cypress-allure-plugin#26]
Hey guys,

My aim is to see screenshots for each test. Well, my code takes the screenshot in a afterEach hook and then I could see the .png added to the allure-results folder as shown in the image below.

![image](https://user-images.githubusercontent.com/42723986/96171932-05fdd200-0efc-11eb-87ac-9f904bc58b92.png)

The allure report opens nicely but the screenshot is not displayed although there is a kind of a section left to that. See below an image of the report:

![image](https://user-images.githubusercontent.com/42723986/96172182-6ab92c80-0efc-11eb-81b9-5ecec34cf8dc.png)

So I tried a workaround by using attachment method of Allure API in order to add it programatically. However, I didn't have any success. 

So, to summarize, it's seems attachment method of Allure API is not working properly.

I added the exact name of the screenshot generated by cypress as cypress documentation: https://docs.cypress.io/api/commands/screenshot.html#Notes
 which worked in a mochawesome report.

Here is the code which I put in index.js under support folder.
```js
import './commands'
import './login.js'
//import addContext from 'mochawesome/addContext'
import '@shelex/cypress-allure-plugin'

Cypress.on('test:after:run', (test, runnable) => {
  if (test.state === 'failed') {
    const screenshotFileName = `${runnable.parent.title} -- ${test.title} (failed).png`
    const path = `../reports/screenshots/${Cypress.spec.name}/${screenshotFileName}`
    cy.allure().attachment('screenshot', new Buffer.from(path,'base64'),'image/png')
    
  }
  else {
    const screenshotFileName = `${runnable.parent.title} -- ${test.title} -- after each hook.png`
    const path = `../reports/screenshots/${Cypress.spec.name}/${screenshotFileName}`
    cy.allure().attachment('screenshot', new Buffer.from(path,'base64'),'image/png')
  }
})
```

In my test I just have a afterEach calling cy.screenshot()
![image](https://user-images.githubusercontent.com/42723986/96173641-9210f900-0efe-11eb-88d8-e42365bc183f.png)

My OS is a Linux Mint 20 Ulyana and allure is in its most recent version I believe which is 2.13.6.

**Comment by Shelex** [comment]
Hi @rodolforodriguess 
Thank you for opening an issue.

1. Why it is not automatically attached:
`afterEach` mocha hook is fired when test itself is finished, and because of that there is no logger for test available in allure at that time and it is trying to add screenshot to current executable, which is afterEach hook logger, and then this hook is attached to suite logger with no screenshot available as allure does not support it. Solution for such issue could be done in terms of #17 with weird idea to handle hooks as steps for test, as order of mocha events [not allow ](https://github.com/mochajs/mocha/issues/4266) to do that natively.

2. Why it cannot be processed in `test:after:run`:
The issue is that cypress internal `test:after:run` event has no relation to mocha runner as according to [documentation](https://docs.cypress.io/api/events/catalog-of-events.html#Cypress-Events) test:after:run fires **after** the test and **all** afterEach and after hooks run. While allure reporter writer itself is an `afterEach` hook, it basically cannot access what is going on in `test:after:run`.

**Comment by Shelex** [comment]
Addressed in v1.10.0
